src/genie/libs/parser/iosxe/tests/ShowPolicyMap/cli/equal/golden_output16_expected.py

```diff
@@ -0,0 +1,16 @@
+expected_output = {
+    'policy_map': {
+        'Policy-Map 1': {
+            'class': {
+                'Class-1': {
+                    'priority': True,
+                    'priority_percent': 18,
+                },
+                'Class-2': {
```

Comment:
Your expected output does not match with the output you shared in txt file. Kindly re-work on it.